templates/webapi/test/infopanel.html.ep

```diff
@@ -166,7 +166,7 @@
                 % }
                 % if (my $scenario_description = $job->scenario_description) {
                     <div id="scenario-description">
-                            %= $scenario_description
+                            %= Mojo::ByteStream->new(OpenQA::Markdown::markdown_to_html($scenario_description))
```

Comment:
Why do we need the outer Mojo::ByteStream?

Reply:
Because otherwise the HTML code would be escaped. However, there's a more elegant way of doing that:

```suggestion
                            %== OpenQA::Markdown::markdown_to_html($scenario_description)
```

But it would also make sense to add a `rendered_scenario_description` function on the job that does this (similar to the `rendered_description` function we already have for job groups).

Reply:
thx for the pointer I will add the `rendered_description`.